.github/workflows/build.yaml

```diff
@@ -0,0 +1,34 @@
+name: Build
+on: [push, pull_request ]
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    env:
+      DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+      DOTNET_CLI_TELEMETRY_OPTOUT: 1
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - run: |
+        pushd src/Tmds.DBus && dotnet restore && dotnet pack -c Release && popd
+        pushd test/Tmds.DBus.Tests && dotnet restore && dotnet test && popd
+        VERSION_SUFFIX="$(date +"%y%m%d")-${{ github.run_id	}}"
+        pushd src/Tmds.DBus.Tool && dotnet restore /p:VersionSuffix="$VERSION_SUFFIX" && dotnet build && popd
+        dotnet pack src/Tmds.DBus --configuration Release --version-suffix "$VERSION_SUFFIX" --output src/Tmds.DBus
+        dotnet pack src/Tmds.DBus.Tool --configuration Release --version-suffix "$VERSION_SUFFIX" --output src/Tmds.DBus.Tool
+
+    - uses: actions/upload-artifact@v2
+      with:
+        name: nuget
+        path: |
+          src/Tmds.DBus/Tmds.Dbus.*.nupkg
+          src/Tmds.DBus.Tool/Tmds.DBus.Tool.*.nupkg
+
+    - run: |
+        if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
```

Comment:
This section is uploading the NuGet package that was build to a myget feed.
We should remove the TRAVIS_BRANCH TRAVIS_PULL_REQUEST condition.

Also, somehow I need to set NUGET_APIKEY in a secure way. I'll have to read the docs for that.

Reply:
I migrated `NUGET_APIKEY` to a secret. You can set it via *Settings*, *Secrets* in the GitHub UI. https://docs.github.com/en/actions/reference/encrypted-secrets has all the details.